overlay/usr/lib/bitkey.d/inithooks-preseed

```diff
@@ -7,5 +7,6 @@ export ROOT_PASS=$MASTERPASS
 export USER_PASS=$MASTERPASS
 export SEC_UPDATES=SKIP
 export HUB_APIKEY=SKIP
+export SEC_ALERTS=SKIP
```

Comment:
Since bitkey is used on a live CD I was not certain that would be useful. Without this option, it is popping a dialog box at every boot. Maybe a mechanism to tell the user to download the latest iso would be interesting when bitkey updates are released.